.gitpod.yml

```diff
@@ -0,0 +1,34 @@
+tasks:
+  - init: npm install && npm run build
+    command: npm run start
+
+vscode:
+  extensions:
+    - dbaeumer.vscode-eslint
+    - esbenp.prettier-vscode
+    - streetsidesoftware.code-spell-checker
+
+ports:
+  - port: 3001
+    onOpen: ignore
+  - port: 4000
```

Comment:
The build selects an available port between 4000 and 4999. It will always be 4000 unless something else is running, though. Not sure if this matters since Gitpod is probably sandboxed, but I figure it's worth pointing out that it's variable.

Reply:
I extended the range, now it's 4000-4999